### ISSUE TITLE
Not working in landscape on iOS7?

### ISSUE THREAD

**Issue by mdpp** [ealeksandrov/EAIntroView#89]
Hi,

I'm probably doing something wrong (or failed to find the correct documentation) but the library doesn't look to work properly in Landscape on iOS7. It does present the view properly in term of dimension but the scroll (and the pages) will stay as the app is in portrait.

Any idea what am I doing wrong?

thanks
Mauro
![tutorial](https://cloud.githubusercontent.com/assets/371985/5674329/d2965020-97a8-11e4-971f-798f569a5277.png)

**Comment by ealeksandrov** [comment]
![screenshot 2015-01-11 21 32 52](https://cloud.githubusercontent.com/assets/3132438/5696256/76a41666-99d9-11e4-8406-c9fddbcf4150.png)
That's working for me when I just enabled landscape mode in demo project. Are you sure that you are handling rotation in parent view controller?

Current support is just a hack, I am working on autolayout and possible solution to handle rotation properly in 3.0 release.

**Comment by ealeksandrov** [comment]
Closed in favour of #79.

**Comment by mdpp** [comment]
Hi,

my app only work in landscape, so should not be a problem of handling the rotation.

I found a way to solve my problem. I used to create the tutorial as son of the main window:
[introTutorial showInView:[[[UIApplication sharedApplication] delegate] window] animateDuration:0.0];
 
I modified (for iOS 7) not using the main window and solved the problem:
[introTutorial showInView:self.view animateDuration:0.0];

Weird.

Best and thanks
Mauro
